packages/connectivity_plus/pubspec.yaml

```diff
@@ -24,23 +24,26 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.0.5
-  connectivity_plus_platform_interface: ^0.4.0
-  connectivity_plus_linux: ^0.3.0
-  connectivity_plus_macos: ^0.4.0
-  connectivity_plus_web: ^0.6.0
-  connectivity_plus_windows: ^0.1.0
+  meta: ^1.3.0-nullsafety.6
+  connectivity_plus_platform_interface: ^0.5.0-nullsafety.0
+  connectivity_plus_linux: ^0.4.0-nullsafety.0
+  connectivity_plus_macos: ^0.5.0-nullsafety.0
+  connectivity_plus_web: ^0.7.0-nullsafety.0
+  connectivity_plus_windows: ^0.2.0-nullsafety.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
   flutter_driver:
     sdk: flutter
   test: any
-  mockito: ^4.1.1
-  plugin_platform_interface: ^1.0.0
-  pedantic: ^1.9.2
+  mockito: ^5.0.0-nullsafety.7
+  plugin_platform_interface: ^1.1.0-nullsafety.2
+  pedantic: ^1.10.0-nullsafety.3
+
+dependency_overrides:
```

Comment:
Added this `args` to `dependency_overrides` (to example/pubspec.yaml as well) because:</br> _Because every version of flutter_driver from sdk depends on args 1.6.0 and every version of dbus from git depends on args ^2.0.0-nullsafety, flutter_driver from sdk is incompatible with dbus from git._


But don't khow to solve that conflict because package is not publishable with `dependency_overrides`

Reply:
I wonder if the whole `args` package dependency was a copy-paste accident or something. I don't see any import of the package, so perhaps we can simply remove it from the pubspec?

Reply:
I think `flutter_driver` dependency can be removed (?) @miquelbeltran  because it seems like the integration test is in the app example dir. This `args` conflict is between `dbus` package and `flutter_driver` package, which have one of them doesn't jave null safety version

Reply:
Yes, indeed. :+1: I don't know how `flutter_driver` ended up as a dependency in so many main packages, but it's only needed/used in the example apps.